test/Subscriber/unsusbcribe.test.ts

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+const sinon = require('sinon');
+import { assert } from 'chai'
+import { createMockClient } from 'mock-apollo-client';
```

Comment:
Why are we using `mock-apollo-client`? 
How does it help in this test?

Reply:
https://www.npmjs.com/package/mock-apollo-client 
Helps unit test components which use the Apollo Client.

Reply:
Please use `sinon.mock` for this.

Reply:
Done!